client/my-sites/plugins-wpcom/premium-plugins-panel.jsx

```diff
@@ -26,9 +30,10 @@ export const PremiumPluginsPanel = React.createClass( {
 
 				<Card className={ cardClasses }>
 					<div className="wpcom-plugins__list">
-						{ plugins.map( ( { name, descriptionLink, icon, plan, description } ) =>
-							<PremiumPlugin
-								{ ...{ name, key: name, descriptionLink, icon, plan, description } }
+						{ plugins.map( ( { name, descriptionLink, icon, category, description } ) =>
+							<Plugin
+								onClick={ () => onClick( name ) }
```

Comment:
We can avoid a fat arrow here, if we move the default analytics tracking down a level to the Plugin component. The Plugin component will have name available as a prop.